### PR TITLE
Adds check to narsie/eldergod summoning, as slaughter summoning already has

### PR DESCRIPTION
**Pull request by Kyep**
Currently, attempting to use the 'call forth the slaughter (demons)' rune as a cultist when you don't have the appropriate objective will punish you, instead of calling the demons. Conversely, attempting to use the summon nar-sie rune when you don't have the correct objective.... still works fine.

This PR makes behavior consistent between the two - such that attempting to use the wrong rune for this critical step will always punish the summoners for doing it wrong. 

🆑 Kyep
tweak: Using the 'summon narsie' rune as cult, when you don't have the objective to do so, is now punished by the cult's god. Just as using the 'call forth the slaughter' rune, when you don't have the objective to do that, is already punished.
/🆑

### CODE REVIEW

**Comment by KingPhilipIII** [comment]
Nar'Sie doesn't like being called into work on his days off does he.

**Comment by Aurorablade** [comment]
i canot remember, if i just forgot to tchange something..but why i left it out was porbaly something to do with adminbus. As in the mode itself you NORMALLY cannot draw the rune without the objective anyway. I do want to punish people who try to summon off z-level however so there is a check for that..

**Comment by Kyep** [comment]
@Aurorablade Yesterday, a group of cultists without the summon objective drew, and successfully used, the summon narsie rune. They had the slaughter objective instead. So, the summon narsie rune does work in some cases where it probably shouldn't - this PR fixes that.

File: code/game/gamemodes/cult/runes.dm [hunk 1]
```diff
@@ -476,6 +476,17 @@ var/list/teleport_runes = list()
...
+	if(!(CULT_ELDERGOD in cult_mode.objectives))
+		message_admins("[usr.real_name]([user.ckey]) tried to summonn an eldritch horror when the objective was wrong")
+		for(var/M in invokers)
+			var/mob/living/L = M
+			to_chat(L, "<span class='cultlarge'><i>\"YOUR SOUL BURNS WITH YOUR ARROGANCE!!!\"</i></span>")
+			if(L.reagents)
+				L.reagents.add_reagent("hell_water", 10)
+			L.Weaken(7)
+			L.Stun(7)
+		fail_invoke()
```
Comment (Fox-McCloud) [inline code/game/gamemodes/cult/runes.dm]: The copy-pastteeeeee

**Comment by Kyep** [comment]
@Fox-McCloud Deleted copypaste (including some other instances of same copypaste, that were in the file before this PR).

**Comment by Fox-McCloud** [comment]
👌

**Pull request merged**